kotlin-testing/src/test/kotlin/com/baeldung/usingspy/CalculatorTests.kt

```diff
@@ -0,0 +1,37 @@
+package com.baeldung.usingspy
+
+import io.mockk.clearMocks
+import io.mockk.every
+import io.mockk.spyk
+import io.mockk.verify
+import junit.framework.Assert.assertEquals
+import org.junit.jupiter.api.Test
+
+class Calculator{
+    fun add(a: Int, b: Int): Int {
+        return a+b
+    }
+    fun findAverage(a: Int, b: Int): Int {
+        val total = add(a,b)
+        return total/2;
+    }
+}
```

Comment:
Let's add a little spacing in here:
```suggestion
class Calculator {
    fun add(a: Int, b: Int): Int {
        return a + b
    }
    fun findAverage(a: Int, b: Int): Int {
        val total = add(a, b)
        return total / 2;
    }
}
```